packages/react-instantsearch-hooks-dom/src/ui/InfiniteHits.tsx

```diff
@@ -3,15 +3,52 @@ import React from 'react';
 import { cx } from './lib/cx';
 
 import type { Hit } from 'instantsearch.js';
-import type { JSXElementConstructor } from 'react';
 
 export type InfiniteHitsProps<THit> = React.ComponentProps<'div'> & {
-  hitComponent?: JSXElementConstructor<{ hit: THit }>;
+  hitComponent?: React.JSXElementConstructor<{ hit: THit }>;
   hits: THit[];
   isFirstPage: boolean;
   isLastPage: boolean;
-  showPrevious?: () => void;
-  showMore: () => void;
+  onShowPrevious?: () => void;
+  onShowMore: () => void;
+  classNames?: Partial<InfiniteHitsClassNames>;
+  translations: InfiniteHitsTranslations;
+};
+
+export type InfiniteHitsClassNames = {
+  /**
+   * Class names to apply to the root element.
```

Comment:
We've been inconsistently addinf periods at the end of descriptions.

We can unify this later though.